docs/Configuring_openshift_virtualization_vm_for_dpdk.md

```diff
@@ -232,6 +232,12 @@ spec:
 
 Once the VM is deployed and running, there are configurations that need to be set once on the guest. The changes are explained on each subchapter with appropriate snippets. In the end, an example guest configuration summary script is offered.
 
+### Linux grub cmdLine on guest
+The hugepages are specified on the grub cmdLine:
+```bash
+grubby --update-kernel=ALL --args="default_hugepagesz=1GB hugepagesz=1G hugepages=8"
```

Comment:
I'm not sure that hugepages=8 is correct, but this is not part of this PR.

Reply:
You are correct. Will need to update in another PR